ansible_rulebook/job_template_runner.py

```diff
@@ -120,7 +120,7 @@ async def run_job_template(
                     counter = event["counter"]
                     if counter not in counters:
                         counters.append(counter)
-                        print(event["stdout"])
+                        logger.debug(event["stdout"])
```

Comment:
@mkanoor do you have the schema of the `event`? aap-eda side needs to parse it

Reply:
@hsong-rh This event is used only for debug logging it is not being sent to the eda-server. This function takes an argument for event_handler which is set to None by default. This loop is still going to run fetching the job events from the controller so that we can detect that the job has finished.